python/build_templates.py

```diff
@@ -109,10 +109,10 @@ def create_template(template, new_filename, replacements):
 
 def main(argv):
     if len(argv) == 1:
-        pkg_path = argv[0]
+        pkg_path = argv[0].rstrip('/')
         pkg = os.path.basename(pkg_path)
     elif len(argv) == 2:
-        pkg_path = argv[0]
+        pkg_path = argv[0].rstrip('/')
         pkg = argv[1]
```

Comment:
Looks fine. However, I suggest you set `pkg_path` outside the if/else so you only have to perform the computation once. The code is identical for both branches.